src/selfie_tools.c

```diff
@@ -21,6 +21,7 @@
 #include "config.h"
 #include "selfie_tools.h"
 #include <sys/time.h>
+#include <time.h>
```

Comment:
Not sure if we need sys/time.h on some systems, but `time()` is defined in `time.h` on fedora28